xml/admin_gui_dashboard.xml

```diff
@@ -137,7 +137,7 @@
   <sect2 xml:id="dashboard-main-menu">
    <title>Main Menu</title>
    <para>
-    The dashboard's main menu occupies the top left part of the screen. It
+    The dashboard's main menu occupies the left-hand side of the screen. It
     covers the following topics:
```

Comment:
I'd start that list with the "landing page" that shows the overall cluster metrics and health status.

Reply:
That is covered in line 213, section: "The Content Pane". Considering when you open the dashboard, you're going to see all of it at once, it's up the individual as to whether or not they want information on the landing page or navigation options.